lib/Chat/ChatManager.php

```diff
@@ -400,6 +400,30 @@ public function getHistory(Room $chat, int $offset, int $limit, bool $includeLas
 		return $this->commentsManager->getForObjectSince('chat', (string) $chat->getId(), $offset, 'desc', $limit, $includeLastKnown);
 	}
 
+	/**
+	 * @param Room $chat
+	 * @param int $offset Last known message id
+	 * @param array $verbs
```

Comment:
Missing description for `ignoreOffset` (I have to admit that it took me a while to understand the parameter and why it was needed :-) ).

Maybe rather than `ignoreOffset` it should be named `secondToPrevious` or something like that? I am not sure of what would be a good name, though.

Reply:
Using `$offsetIsVerbMatch` now and inverted the logic